api/views/andela_centres.py

```diff
@@ -33,7 +36,7 @@ class CountryViewset(ModelViewSet):
 class AndelaCentreViewset(ModelViewSet):
     serializer_class = AndelaCentreSerializer
     queryset = models.AndelaCentre.objects.all()
-    permission_classes = [IsAuthenticated, IsAdminUser]
+    permission_classes = [IsAuthenticated, IsAdminUser, IsAdminReadOnly | IsSuperAdmin]
```

Comment:
Please change this to (IsAdminReadOnly | IsSuperAdmin,)

Reply:
@egichuri I remove the other authentication classes?